platform/linux-generic/odp_schedule_basic.c

```diff
@@ -1658,13 +1754,13 @@ static int schedule_multi_no_wait(odp_queue_t *out_queue, odp_event_t events[],
 static int schedule_multi_wait(odp_queue_t *out_queue, odp_event_t events[],
 			       int num)
 {
-	int ret;
-
-	do {
-		ret = schedule_run(out_queue, events, num);
-	} while (ret == 0);
+	return schedule_loop(out_queue, ODP_SCHED_WAIT, events, num);
+}
```

Comment:
This is actually just a leftover from a previous version. Would be nice to fix that little eyesore, but it could potentially cost a few cycles. Anyway, this change is not needed in this PR, removed in v3.